pkg/limayaml/limayaml.go

```diff
@@ -45,6 +45,11 @@ type File struct {
 type Mount struct {
 	Location string `yaml:"location" json:"location"` // REQUIRED
 	Writable bool   `yaml:"writable,omitempty" json:"writable,omitempty"`
+	Sshfs    Sshfs  `yaml:"sshfs,omitempty" json:"sshfs,omitempty"`
+}
+
+type Sshfs struct {
+	FollowSymlinks bool `yaml:"follow_symlinks,omitempty" json:"follow_symlinks,omitempty"`
```

Comment:
The json/yaml name should be `followSymlinks` to match the style used by other fields.